js/instructor.js

```diff
@@ -0,0 +1,83 @@
+const instructorsCards = document.querySelector('.instructor-main');
+const moreButton = document.querySelector('.more');
+const instructors = [
+  {
+    name: 'Alexei Kovalev',
+    bio: ' British software engineer and the creator of the open-source NoSQL database system Couchbase',
+    brief: "Kovalev's work has contributed to the development of high-performance, scalable database systems",
+    image: './images/instructor/instructor_1.svg',
+  },
+  {
+    name: 'Guido van Rossum',
+    bio: 'Dutch programmer and creator of the Python programming language',
+    brief: 'Van Rossum is known for his contributions to open source software and has worked for companies including Google and Dropbox',
+    image: './images/instructor/instructor_2.svg',
+  },
+  {
+    name: 'Susan Wojcicki',
+    bio: 'American technology executive and the current CEO of YouTube',
+    brief: 'Wojcicki has made significant contributions to the development of online video and has worked for companies including Google and Intel.',
+    image: './images/instructor/instructor_3.svg',
+  },
+  {
+    name: 'Grace Hopper',
+    bio: 'American businesswoman and former CEO of Yahoo!',
+    brief: 'Hopper has made significant contributions to the development of internet search and online advertising.',
+    image: './images/instructor/instructor_4.svg',
+  },
+  {
+    name: 'Michael Widenius',
+    bio: ' Finnish software engineer and the original developer of the MySQL database management system',
+    brief: 'Widenius is known for his contributions to the development of open source software and has founded several successful software companies',
+    image: './images/instructor/instructor_5.svg',
+  },
+  {
+    name: 'Martin Fowler',
+    bio: 'British software developer, author, and international speaker on software development topics',
+    brief: 'Fowler is known for his work on software design patterns and agile software development methodologies.',
+    image: './images/instructor/instructor_6.svg',
+  },
+];
+
+function buildInstructorCard(index) {
+  const buildCardHTML = `
+        <li class="instructor-item" >
+            <img src="${instructors[index].image}" alt="${instructors[index].name}'s image" class="instructor-img">
```

Comment:
- [ ] Please 🙏🏽 use the `alt` attribute for the image for the best practices of HTML & CSS.

Reply:
@franclobo 
you requested the addition of an alt attribute to certain img tag, but it appears that alt attributes already exist.

![WhatsApp Image 2023-05-24 at 6 53 08 PM](https://github.com/ahmedeid6842/Capstone/assets/57197702/4ed07850-772e-43bb-aaba-7b05ca354e8e)